### PR TITLE
tests: add example of secondary interface multi-zone deployment

### PR DESCRIPTION
**Pull request by jor2**
### Description

Added an example for multi-zone deployment with secondary interface configured.

https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/issues/518

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- When merging, use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

**Comment by jor2** [comment]
Needed to skip upgrade test due to prefix being duplicated in `slz_vpc` module as we pass prefix value for name and prefix which are both used in naming convention resulting in characters being exceeded in test and failing. https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/pull/547/files#diff-07f337e38ed1996f30ee4f04357ab23318c254cb606aaf4745d564cbd546722dR67

File: examples/complete/main.tf [hunk 1]
```diff
@@ -64,7 +64,7 @@ module "slz_vpc" {
   region            = var.region
   prefix            = var.prefix
   tags              = var.resource_tags
-  name              = "${var.prefix}-vpc"
+  name              = "vpc"
```
Comment (jor2) [inline examples/complete/main.tf:67]: was duplicating prefix name, resulting in name being too long.

Comment (ocofaigh) [reply examples/complete/main.tf:67]: how about just setting it to var.prefix ?

File: examples/complete/README.md [hunk 2]
```diff
@@ -7,4 +7,5 @@ It will provision the following:
 - A new VPC with 3 subnets.
 - A new placement group.
 - A VSI in each subnet placed in the placement group.
+- A secondary VSI with secondary subnets and secondary security group
```
Comment (ocofaigh) [inline examples/complete/README.md:10]: Lets also update the title of the readme here to mention this

**Review by ocofaigh — COMMENTED** [review]
@jor2 looks good - some minor comments

File: examples/complete/main.tf (hunk 1 above)
Comment (jor2) [reply examples/complete/main.tf:67]: It's coming from here https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/blob/118d05d9d51c4097a25d4bd17336e268b62c64de/main.tf#L104

**Review by ocofaigh — APPROVED** [review]

**Comment by ocofaigh** [comment]
Pipeline failed with:
```
│ Error:  failed to Destroy key ring with error: kp.Error: correlation_id='7bc220ed-fa1b-4dd9-89b1-d8de4f81e677', msg='Conflict: Key ring could not be deleted: Please see `reasons` for more details (KEY_RING_NOT_EMPTY_ERR)', reasons='[KEY_RING_NOT_EMPTY_ERR: The specified key ring contains at least one key (in any state) - FOR_MORE_INFO_REFER: https://cloud.ibm.com/apidocs/key-protect]'
│ 
│ 
╵}
```

re-running ..

**Comment by ocofaigh** [comment]
Same issue reproduced - lets try again on Monday

**Comment by ocofaigh** [comment]
@jor2 Its possible there is a bug in the complete example causing this, can you check?

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**